dali/operators/audio/nonsilence_op.cc

```diff
@@ -67,10 +67,11 @@ bool NonsilenceOperatorCpu::SetupImpl(std::vector<OutputDesc> &output_desc,
   TypeInfo output_type;
   output_type.SetType<int>(TypeTable::GetTypeID<int>());
   TensorShape<> scalar_shape = {};
+  auto curr_batch_size = ws.get_input_batch_size(0);
```

Comment:
As I mentioned by this function, it would be nice to have a variant without arguments, that would return the common batch size and enforce that all inputs use the same one. With 1 input it would do the same as this call, but we could drop the 0 - most of our ops assume equal batches in all inputs.

Reply:
In an offline discussion we decided, that it's not necessary to add `GetInputBatchSize()` overload